balancer-js/src/modules/pools/factory/composable-stable/composable-stable.factory.integration.spec.ts

```diff
@@ -0,0 +1,180 @@
+// yarn test:only ./src/modules/pools/factory/composable-stable/composable-stable.factory.integration.spec.ts
+import { OldBigNumber, StableMaths } from '@balancer-labs/sor';
+import { Interface, LogDescription } from '@ethersproject/abi';
+import { parseFixed } from '@ethersproject/bignumber';
+import { AddressZero } from '@ethersproject/constants';
+import { Contract } from '@ethersproject/contracts';
+import { expect } from 'chai';
+import dotenv from 'dotenv';
+import { ethers } from 'hardhat';
+
+import { Vault__factory } from '@/contracts/factories/Vault__factory';
+import ComposableStablePoolAbi from '@/lib/abi/ComposableStable.json';
```

Comment:
Lets update the Typechain to generate the types version of these contracts and use the factories.
(Let me know if you need help with this)

Reply:
Bruno helped me with that, thanks!